ts/features/bonus/cgn/components/merchants/CgnDiscountDetail.tsx

```diff
@@ -78,19 +79,22 @@ const CgnDiscountDetail: React.FunctionComponent<Props> = ({
   return (
     <View style={styles.container}>
       {index(0, [...discount.productCategories]).fold(undefined, categoryKey =>
-        getCategorySpecs(categoryKey).fold(undefined, c => (
-          <View style={styles.row}>
-            <IconFont
-              name={c.icon}
-              size={CATEGORY_ICON_SIZE}
-              color={IOColors.bluegrey}
-            />
-            <View hspacer small />
-            <H5 weight={"SemiBold"} color={"bluegrey"}>
-              {I18n.t(c.nameKey).toLocaleUpperCase()}
-            </H5>
-          </View>
-        ))
+        getCategorySpecs(categoryKey as ProductCategoryEnum).fold(
```

Comment:
```suggestion
        getCategorySpecs(categoryKey).fold(
```
This should not be required, in my understanding the type seems already inferred. What you think?

Reply:
I handled it cause it was in the output of `tsc:noemit` check...seems not to be required anymore

Reply:
ddde264 fixes